java/core/src/main/java/co/worklytics/psoxy/rules/jira/PrebuiltSanitizerRules.java

```diff
@@ -0,0 +1,589 @@
+                    .jsonPath("$..displayName")
+                    .build())
+            .transform(Transform.Pseudonymize.builder()
+                    .jsonPath("$..accountId")
+                    .jsonPath("$..emailAddress")
+                    .build())
+            .build();
+
+    static final Endpoint GROUP_BULK = Endpoint.builder()
+            .pathTemplate("/ex/jira/{cloudId}/rest/api/{apiVersion}/group/bulk")
+            .allowedQueryParams(commonAllowedQueryParameters)
+            .transform(Transform.Redact.builder()
+                    .jsonPath("$..name")
+                    .build())
+            .build();
+
+    static final Endpoint GROUP_MEMBER = Endpoint.builder()
+            .pathTemplate("/ex/jira/{cloudId}/rest/api/{apiVersion}/group/member")
+            .allowedQueryParams(groupMemberAllowedQueryParameters)
+            .transform(Transform.Redact.builder()
+                    .jsonPath("$.values[*].self")
+                    .jsonPath("$.values[*].avatarUrls")
+                    .jsonPath("$.values[*].displayName")
+                    .jsonPath("$.values[*].name")
+                    .build())
+            .transform(Transform.Pseudonymize.builder()
+                    .jsonPath("$.values[*].accountId")
+                    .jsonPath("$.values[*].emailAddress")
+                    .build())
+            .build();
+
+    static final Endpoint PROJECTS = Endpoint.builder()
+            .pathTemplate("/ex/jira/{cloudId}/rest/api/{apiVersion}/project/search")
+            .allowedQueryParams(commonAllowedQueryParameters)
+            .transform(Transform.Redact.builder()
+                    .jsonPath("$.values[*]..avatarUrls")
+                    .jsonPath("$.values[*]..self")
+                    .jsonPath("$.values[*]..displayName")
+                    .jsonPath("$.values[*]..leadUserName")
+                    .jsonPath("$.values[*]..description")
+                    .build())
+            .transform(Transform.Pseudonymize.builder()
+                    .jsonPath("$.values[*]..accountId")
+                    .jsonPath("$.values[*]..emailAddress")
+                    // An email associated with the project
+                    .jsonPath("$.values[*]..email")
+                    .jsonPath("$.values[*]..leadAccountId")
+                    .build())
+            .build();
+
+    @VisibleForTesting
+    static final RESTRules JIRA_CLOUD = Rules2.builder()
+            .endpoint(GROUP_BULK)
+            .endpoint(GROUP_MEMBER)
+            .endpoint(ISSUE_SEARCH_V2)
+            .endpoint(ISSUE_SEARCH_V3)
+            .endpoint(ISSUE_CHANGELOG)
+            .endpoint(ISSUE_COMMENT_V2)
+            .endpoint(ISSUE_COMMENT_V3)
+            .endpoint(ISSUE_WORKLOG_V2)
+            .endpoint(ISSUE_WORKLOG_V3)
+            .endpoint(USERS)
+            .endpoint(PROJECTS)
+            .build();
+
+    @VisibleForTesting
+    static final RESTRules JIRA_SERVER = Rules2.builder()
```

Comment:
Two group of rules: one for cloud and another one for server. Server doesn't have same version of api than cloud (cloud is 3 in beta, server in 2) so:
- Adding rules for cloud for version 2 and 3 when required; main difference is with [comments](https://developer.atlassian.com/cloud/jira/platform/rest/v3/intro/#version) where in 3 we could extract information about mentions, etc. But in 2, comment appears *as-is*, so it should be redacted. In 3, is pseudonymized.
- Groups, Users, Projects and Changelogs are only present for Cloud, not for Server.

For that, there are test files for both versions (2,3 and server/cloud) and two test classes to cover them.